extensions/amp-recaptcha-input/0.1/amp-recaptcha-input.js

```diff
@@ -146,6 +151,27 @@ export class AmpRecaptchaInput extends AMP.BaseElement {
         ' and data-action attribute'
     ));
   }
+
+  /**
+   * Function to check if recaptcha experiment is enabled,
+   * through origin trial, or AMP.toggleExperiment
+   * @return {!Promise<boolean>}
+   */
+  _isExperimentEnabled() {
```

Comment:
isExperimentEnabled_

Reply:
Oops, thanks for catching that. I do the _ in front of personal fields in personal projects. 😂